charts/seed-monitoring/charts/grafana/dashboards/owners/node-pool-dashboard.json

```diff
@@ -897,6 +897,115 @@
       "transform": "table",
       "type": "table-old"
     },
+    {
+      "aliasColors": {
+      },
+      "bars": false,
+      "dashes": false,
+      "dashLength": 10,
+      "datasource": null,
+      "decimals": 1,
+      "description": "Shows the number of node that are in the NotReady state for more than 10 minutes du to the kubelet not responding",
+      "fieldConfig": {
+          "defaults": {
+          },
+          "overrides": [
+          ]
+      },
+      "fill": 0,
+      "fillGradient": 0,
+      "gridPos": {
+          "h": 7,
+          "w": 12,
+          "x": 12,
+          "y": 32
+      },
+      "hiddenSeries": false,
+      "id": 21,
+      "legend": {
+          "alignAsTable": false,
+          "avg": false,
+          "current": false,
+          "max": false,
+          "min": false,
+          "show": true,
+          "total": false,
+          "values": false
+      },
+      "lines": true,
+      "linewidth": 1,
+      "links": [
+      ],
+      "nullPointMode": "null",
+      "options": {
+          "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "7.5.17",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [
+      ],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+          {
+              "exemplar": true,
+              "expr": "count_over_time(\n  sum by (node, type)(last_over_time(kube_node_status_condition{condition=\"Ready\",status!=\"true\"}[1m]) == 1)[10m:1m]\n) >= 10",
```

Comment:
see above

Reply:
Expression fixed (commented and detailed above)